vendor/licenses/git_submodule/apex-tmLanguage.dep.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: apex-tmLanguage
+version: 77279761696d92deb26921210f20786bf8f3ffbc
+type: git_submodule
+homepage: https://github.com/forcedotcom/apex-tmLanguage.git
+license: other
+licenses:
+- sources: LICENSE
+  text: |
+    License for modifications and not otherwise marked:
```

Comment:
🤔 what is this license? It needs to be one of [there](https://github.com/github/linguist/blob/9b1023ed5d308cb3363a882531dea1e272b59977/vendor/licenses/config.yml#L4-L15) in order for us to accept it.

Reply:
Looks like the original contributor added the [new BSD-3](https://opensource.org/license/bsd-3-clause/) above of the MIT. 😅

I'll ask the team to update it properly. Given that both licenses fit the requirement, is this an immediate blocker?

Reply:
Perfect! This isn't a blocker. We just need to change the `other` to `bsd-3-clause`.